hyde-themes.db

```diff
@@ -86,3 +86,23 @@ DESCRIPTION:""
 LINK="https://github.com/prasanthrangan/hyde-themes/tree/Rose-Pine"
 OWNER="https://github.com/prasanthrangan"
 DESCRIPTION:""
+
+[AI Arts]
+LINK="https://github.com/tryprncp/Themes/tree/AI-Arts"
+OWNER="https://github.com/tryprncp"
+DESCRIPTION:""
+
+[Abstract]
```

Comment:
https://github.com/kRHYME7/hyde-gallery/blob/hyde-gallery/Windows%2011/Windows%252011.png

You can use the Windows 11 theme for this I guess should look better 

Here's the repo
https://github.com/kRHYME7/hyde-gallery/tree/Windows-11